pkgs/applications/networking/sniffers/savvycan/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, qtbase
+, qttools
+, qmake
+, qtserialbus
+, qtserialport
+, qtdeclarative
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "SavvyCAN";
```

Comment:
```suggestion
  pname = "savvycan";
```

Reply:
Those were probably added before the rule was introduced, or are part of an autogenerated package set, or require capitalisation for a technical reason.

https://nixos.org/manual/nixpkgs/stable/index.html#sec-package-naming

> The pname and the version attribute must not contain uppercase letters — e.g., "mplayer" instead of “MPlayer”`

I think `savvy-can` would be acceptable too

> Dashes in the package pname should be preserved in new variable names, rather than converted to underscores or camel cased — e.g., http-parser instead of http_parser or httpParser. The hyphenated style is preferred in all three package names

But [other distributions use `savvycan`](https://repology.org/project/savvycan/versions) so it's better to be consistent